### DISCUSSION TITLE
希望视频逐帧调节可以每调一下都自动暂停

### DISCUSSION THREAD

**Discussion by LOVE-END** [the1812/Bilibili-Evolved#2588] [功能建议 / Ideas]
emmm,因为逐帧调节就是想看到一些只有几帧的画面嘛
调完以后没有自动暂停,视频还会继续播放,调的还没有播放的快
暂停的时候调节一下也会继续播放
希望作者大大可以优化一下呀~
超级感谢

**Comment by the1812** [comment]
不会继续播放啊

**Reply by Geeyun-JY3** [reply]
楼主的意思应该是希望在视频播放时逐帧调节可以使视频自动暂停（即暂停了再调？）

**Reply by LOVE-END** [reply]
> 不会继续播放啊

噢噢,确实不会,是我的问题~哈哈哈哈,不知道为什么上次这样操作失败了
emm,谢谢作者大大,问题解决啦

**Reply by Geeyun-JY3** [reply]
暂停时调节确实不会播放，但播放时调节好像确实还会继续播放啊（楼主不是还要播放时调节自动暂停吗）
